### ISSUE TITLE
Unresolved references, network/loss.py not used

### ISSUE THREAD

**Issue by redsphinx** [uw-ipd/RoseTTAFold2NA#28]
I want to finetune the model on my own data and some references are missing. Some examples: 

- `network/loss.py`: the functions are not called anywhere
- `network/ffindex.py`: Unresolved reference 'read_lines'
- `network/parsers.py`: Unresolved reference 'L_s', Unresolved reference 'FFindexDB'

Where is the `loss.py` code actually used?

**Comment by fdimaio** [comment]
Hello, loss.py is part of training.  We have not yet included our training code, but can add it in the future.

**Comment by redsphinx** [comment]
Do you have an estimate for when the training code could be uploaded? Or any other resource I can look to for training/fine-tuning my own model?

**Comment by luukhd2** [comment]
Hi all,
I have the same issue, could the training code be uploaded, or sent privately if it is not fully ready?
If not, is there an approach you recommend for looking at fine-tuning the pre-trained model?

**Comment by redsphinx** [comment]
Hey guys, I have another question related to the loss.py. In the paper you outline mutiple loss functions. I can't seem to find the seq loss (masked amino-acid recovery loss)
